Userland/Libraries/LibWeb/CSS/Default.css

```diff
@@ -747,6 +747,10 @@ video {
     object-fit: contain;
 }
 
+audio:not([controls]) {
+    display: none !important;
+}
```

Comment:
Please quote the spec text here, as you did in the commit message. That'll make it clearer that it's based on the text, not on a spec CSS snippet.

Reply:
It is same web-spec, I just confirmed, so I moved in already defined section. :-)

As for test, I don't have local build of ladybird so I don't know whether running test script will make it work. :-(

Reply:
Please do not open PRs for changes you haven't first built and tested.